changelog.txt

```diff
@@ -14,6 +14,7 @@
 * Add - Disable payment gateway when not in test mode and not using https or ssl checkout enforcement.
 * Fix - Improved errors handling during onboarding and page overview.
 * Update - Remove Account in the old Settings page.
+* Add - Show fee breakup in transaction details timeline.
```

Comment:
Here and `readme.txt`

```diff
-breakup
+breakdown
```

Reply:
Sorry missed it, fixed it in: https://github.com/Automattic/woocommerce-payments/pull/2500/commits/9ab0d164cfb93e2d434530b688b2cf9c70a2f805